private_gpt/ui/ui.py

```diff
@@ -409,11 +409,55 @@ def _build_ui_blocks(self) -> gr.Blocks:
                         inputs=system_prompt_input,
                     )
 
+                    def get_model_label() -> str:
+                        """Get model label from llm mode setting YAML.
+
+                        Raises:
+                            ValueError: If an invalid 'llm_mode' is encountered.
+
+                        Returns:
+                            str: The corresponding model label.
+                        """
+                        # Get model label from llm mode setting YAML
+                        # Labels: local, openai, openailike, sagemaker, mock, ollama
+                        config_settings = settings()
+                        if config_settings is None:
+                            raise ValueError("Settings are not configured.")
+
+                        # Get llm_mode from settings
+                        llm_mode = config_settings.llm.mode
+
+                        # Mapping of 'llm_mode' to corresponding model labels
+                        model_mapping = {
+                            "local": config_settings.local.llm_hf_model_file,
```

Comment:
There is no "local" anymore. It has been changed to "llamacpp".

Reply:
I've updated the code to reflect the name change https://github.com/imartinez/privateGPT/pull/1648/commits/84590252609ddc623e572bd9af55954d21b2cbd9